packages/isomorphic/src/internal/normInput.ts

```diff
@@ -0,0 +1,5 @@
+import { Input } from './index'
+
+export default function normInput(input: Input) {
```

Comment:
All of these files need docstrings - they don't need to be as detailed as external documentation, but I as a package dev should be able to tell what the functions are doing, and right now I don't really.